src/client/src/apps/MainRoute/widgets/analytics/epics/epics.test.ts

```diff
@@ -36,7 +36,9 @@ describe('Analytics epics', () => {
       const actionlifteTime = '-a-a-(rs)a--'
       const expecteLifetime = '-----a--'
 
-      const loadBalancedServiceStub = new MockServiceStubWithLoadBalancer(createStreamOperation$)
+      const loadBalancedServiceStub: ServiceStubWithLoadBalancer = new MockServiceStubWithLoadBalancer(
```

Comment:
I am very ashamed of this typing horror, but for now, this is the quickest resolution that I found to this type problem